stdlib/3/xmlrpc/server.pyi

```diff
@@ -7,9 +7,19 @@ from xmlrpc.client import Fault
 from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Pattern, Protocol, Tuple, Type, Union
 from datetime import datetime
 
-_Marshallable = Union[None, bool, int, float, str, bytes, tuple, list, dict, datetime]
-class _DispatchProtocol(Protocol):
+_Marshallable = Union[None, bool, int, float, str, bytes, tuple, list, dict, datetime]  # TODO: Recursive type on tuple, list, dict
+
+# The dispatch accepts anywhere from 0 to N arguments, no easy way to allow this in mypy
+class _DispatchArity0(Protocol):
+    def __call__(self) -> _Marshallable: ...
+class _DispatchArity1(Protocol):
+    def __call__(self, __arg1: _Marshallable) -> _Marshallable: ...
+class _DispatchArity2(Protocol):
+    def __call__(self, __arg1: _Marshallable, __arg2: _Marshallable) -> _Marshallable: ...
```

Comment:
I don't really know xmlrpc, but how likely is it that somebody would patch a fixed-arity dispatcher with more than two arguments? Maybe to be safe we should add arity 3 and 4 versions.

Reply:
I'm honestly not sure the likelihood. I'll add the 3/4 arity versions, as it doesn't hurt to do so